docs/standard/microservices-architecture/multi-container-microservice-net-applications/implementing-api-gateways-with-ocelot.md

```diff
@@ -0,0 +1,566 @@
+---
```

Comment:
remove ing verbs from file name and remove words that don't add SEO value such as articles, with, etc.
so file name could be implement-api-gateways-ocelot

See rules [here](https://review.docs.microsoft.com/en-us/help/contribute/contribute-azure-file-names-and-locations?branch=master#rules)

Reply:
I would like to make an exception for this particular case (Ocelot). In this case, I don't want to be so prescriptive. Ocelot is just one approach.
If I say "Implement API Gateways with Ocelot" it sounds like we just recommend to do it with Ocelot. Don't you think so?
But other approaches like Azure API Management or additional third parties could also apply..